cross/arm/toolchain.cmake

```diff
@@ -0,0 +1,28 @@
+set(CROSS_ROOTFS $ENV{ROOTFS_DIR})
+
+set(CMAKE_SYSTEM_NAME Linux)
```

Comment:
If you request a cross build, should we display a warning and disable it if you're not on a supported system? e.g. what happens if someone tries to specify cross on OS X?